oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/LogoutService.java

```diff
@@ -174,6 +174,32 @@ private APIGatewayProxyResponseEvent generateAccountInterventionLogoutResponse(
             Optional<String> sessionId,
             AccountInterventionStatus accountStatus) {
         String redirectUri;
+
+        if (configurationService.getEnvironment() == "staging") {
+            if (accountStatus.blocked()) {
+                redirectUri = "/unavailable-permanent";
+                LOG.info("Generating Account Intervention blocked logout response");
+            } else if (accountStatus.suspended()) {
+                redirectUri = "/unavailable-temporary";
+                LOG.info("Generating Account Intervention suspended logout response");
+            } else {
+                throw new RuntimeException("Account status must be blocked or suspended");
+            }
+            sessionId.ifPresent(
+                    t ->
+                            cloudwatchMetricsService.incrementLogout(
+                                    clientId, Optional.of(accountStatus)));
+
+            return generateLogoutResponse(
+                    ConstructUriHelper.buildURI(
+                            "https://signin.staging.account.gov.uk", redirectUri),
```

Comment:
This needs to be environment specific. Once it's in production I would mind hardcoding the prod URL, but something like this is too easy to miss I think